### ISSUE TITLE
Show Feat and Action Traits in Chat Message

### ISSUE THREAD

**Issue by Vaizt** [foundryvtt/pf2e#2378]
Currently when you post an action or feat to chat it does not have the accompanying traits, such as Press, Flourish, Aasimar, etc. This often leads to confusion or misleading ideas of how things work, as in an instance of a monk at my group's table using Combat Grab as their opening action while it has the Press trait and the GM not noticing as we more often than not post our actions to the chat for the ease of others not having to open our trait.

Posting the traits along with the actions or feats allows those with a better understanding of the game, in the case of players not realizing what a trait does, or as a reminder that other players or the GM can bring up to that player rather than having to go look at the feat itself after already being presented it in one form. 

tl;dr: Traits not posting with actions or feats can be confusing to other players, lead to unintended uses of actions or feats and I would absolutely love to see this added just as a Quality of Life addition.
